docs/guide/warnings.rst

```diff
@@ -980,6 +980,12 @@ List Of Warnings
    be declared before being used.
 
 
+.. option:: PRAGMAERR
+
+   Any error that is related to pragmas, but not specifically to '`pragma protect'.
+   For example, an empty `pragma line.
```

Comment:
```suggestion
.. option:: BADPRAGMA

   Error that a pragma is badly formed, but not specifically to '`pragma protect'.
   For example, an empty `pragma line.
   
```

(And re-sort).  Generally don't want ERR in names as they can be changed into warnings a warning with ERR in the name would be confusing.